sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/PhpUnitItCoverageResultImporter.java

```diff
@@ -23,15 +23,9 @@
 import org.sonar.api.batch.sensor.coverage.CoverageType;
 import org.sonar.api.measures.CoreMetrics;
 
-/**
- * The Class PhpUnitItCoverageResultParser.
- */
-public class PhpUnitItCoverageResultParser extends PhpUnitCoverageResultParser {
+public class PhpUnitItCoverageResultImporter extends PhpUnitCoverageResultImporter {
 
-  /**
-   * Instantiates a new php unit coverage result parser.
-   */
-  public PhpUnitItCoverageResultParser(FileSystem fileSystem) {
+  public PhpUnitItCoverageResultImporter(FileSystem fileSystem) {
     super(fileSystem);
     linesToCoverMetric = CoreMetrics.IT_LINES_TO_COVER;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "IT_LINES_TO_COVER"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)